python quiz question.

```diff
@@ -0,0 +1,9 @@
+ {
+    Question: "Which of the following is the best way to input n spaced intigers in python(eg. 1 2 3 4 5).?",
+    Answer: "Lis1 = list(map(int, input().split()))",
+    Distractor1: "Lis1 = input().split()",
+    Distractor2: "Lis1 = int(input())",
+    Distractor3: "Lis3 = list(map(int, input().split()))",
+    Explaination: map function in python makes the items of a list directly into intiger form while in input().split() the intigers will be added in string format.
```

Comment:
Let's just provide an explanation for the map function. If users want to learn more, then they can click on the learn more link in the article. 👍 
Also you have some syntax errors here because of the missing quotes. 
```suggestion
    Explanation: "The map() function is used to apply a function to each item in an iterable (like a Python list or dictionary).",
```